include/wlr/render/dmabuf.h

```diff
@@ -0,0 +1,29 @@
+#ifndef WLR_RENDER_DMABUF_H
+#define WLR_RENDER_DMABUF_H
+
+// So we don't have to pull in linux specific drm headers
+#ifndef DRM_FORMAT_MOD_INVALID
+#define DRM_FORMAT_MOD_INVALID ((1ULL<<56) - 1)
+#endif
+
+#define WLR_DMABUF_MAX_PLANES 4
+
+enum wlr_dmabuf_attributes_flags {
+	WLR_DMABUF_BUFFER_ATTRIBS_FLAGS_Y_INVERT = 1,
+	WLR_DMABUF_BUFFER_ATTRIBS_FLAGS_INTERLACED = 2,
+	WLR_DMABUF_BUFFER_ATTRIBS_FLAGS_BOTTOM_FIRST = 4,
+};
```

Comment:
When the _buffer_ is dropped from the struct name (which is fine since we're now talking about a dmabuf not the buffer of the dmabuf wayland extensions) the _BUFFER_ should IMHO be dropped here as well. Otherwise it looks fine.

Reply:
Ah, forgot this, thanks